src/utils.js

```diff
@@ -51,4 +51,32 @@ const toJson = str => {
   str = str.trim();
   return JSON.parse(str);
 };
-export default { getSortColumnAndDirection, filterTable, sortTable, isJson, toJson };
+
+const timeUnitsBetween = (startDate, endDate) => {
```

Comment:
I understood the factor thing. But it's weird. 

I think it would be best to name this formatDuration(durationInMiliseconds).

I would be easier to userstand `formatDuration(`(new Date() - startedDatetime) * factor)` right ?